syft/workers/base.py

```diff
@@ -79,7 +79,16 @@ def __init__(
         """Initializes a BaseWorker."""
         super().__init__()
         self.hook = hook
-        self.torch = None if hook is None else hook.torch
+        if hook is None:
+            self.framework = None
+        else:
+            # TODO[jvmancuso]: avoid branching here if possible, maybe by changing code in
```

Comment:
Please create an issue for this and link it here

Reply:
https://github.com/OpenMined/PySyft/issues/2530